integration/container/inspect_test.go

```diff
@@ -68,3 +70,35 @@ func TestInspectAnnotations(t *testing.T) {
 	assert.NilError(t, err)
 	assert.Check(t, is.DeepEqual(inspect.HostConfig.Annotations, annotations))
 }
+
+// TestNetworkAliasesAreEmpty verifies that network-scoped aliases are not set
+// for non-custom networks (network-scoped aliases are only supported for
+// custom networks, except for the "Default Switch" network on Windows).
+func TestNetworkAliasesAreEmpty(t *testing.T) {
+	ctx := setupTest(t)
+	apiClient := request.NewAPIClient(t)
+
+	netModes := []string{"host", "bridge", "none"}
+	if runtime.GOOS == "windows" {
+		netModes = []string{"nat", "none"}
+	}
+
+	for _, nwMode := range netModes {
+		t.Run(nwMode, func(t *testing.T) {
+			ctr := container.Create(ctx, t, apiClient,
+				container.WithName("ctr-"+nwMode),
+				container.WithImage("busybox:latest"),
+				// This command is invalid on Windows but since we never start the container, it doesn't really matter.
```

Comment:
IF we don’t need a command, should we just remove the `container.WithCmd("top")` ? (busybox has a default.. `/bin/sh`)